test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -0,0 +1,108 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2017.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#include "ucp_test.h"
+#include <common/test_helpers.h>
+#include <ifaddrs.h>
+#include <ucs/sys/sys.h>
+
+class test_ucp_sockaddr : public ucp_test {
+public:
+    static ucp_params_t get_ctx_params() {
+        ucp_params_t params = ucp_test::get_ctx_params();
+        params.features |= UCP_FEATURE_TAG;
+        return params;
+    }
+
+    virtual void init() {
+        UCS_TEST_SKIP_R("Functions aren't implemented yet");
```

Comment:
why add all this code if it's not running?

Reply:
as preparation for the next prs. this file is more than 100 lines so adding it to other client-server prs will be very difficult...